docs/source/deployment/now.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Deploying with Zeit Now
+sidebar_title: Now
+description: Deploying your GraphQL server to Zeit Now
+---
+
+`now` is a service by Zeit that allows you to deploy an instance of Apollo Server and have a functional GraphQL endpoint. `now` supports three types of deployments for web apps: `static`, `node.js`, and `Docker`. We'll focus on the Node.js deployment for this guide.
```

Comment:
@abernix has been pushing to not use 1st/2nd person pronouns (I, me, my, you, your) in docs. He described more here: https://github.com/apollographql/apollo/pull/169

Reply:
Thanks for this @JakeDawkins